src/strategies/pob-hash-cv/index.ts

```diff
@@ -0,0 +1,55 @@
+import { subgraphRequest } from '../../utils';
+
+export const author = 'bonustrack';
+export const version = '0.1.0';
+
+export const SUBGRAPH_URL = {
+  '1':
+    'https://api.thegraph.com/subgraphs/name/proofofbeauty/subgraph',
+};
+
+export async function strategy(
+  _space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const hashOwnersParams = {
+    hashOwners: {
+      __args: {
+        where: {
+          id_in: addresses.map(a => a.toLowerCase()),
+        },
+        first: 1000, // IS THIS ENOUGH?
```

Comment:
This is enough if there is less than 1000 voters (that's often the case), you would need to paginate request to support more than 1000 voters using `first: 1000, skip: 1000`